### ISSUE TITLE
Release Maven artifact

### ISSUE THREAD

**Issue by pires** [noskovd/spring-data-orient#5]
Releases and eventually snapshots should be made publicly available to Maven central repository.

**Comment by trainings** [comment]
I'll try to implement.

**Comment by pires** [comment]
I can do this but I'm not sure if the current artifact will be accepted, since it's managed by the Spring guys. If you give me a new package, I'll do it.

**Comment by wendymcameron** [comment]
Yes I am also looking for Maven artefact when will one be available as the work I am doing means that I don't currently have local repository.

**Comment by sleroy** [comment]
I am planning to upload a version on bintray/jcenter.

**Comment by mkalus** [comment]
Any update on the current status of a Maven repository?
